app/controllers/projects_controller.rb

```diff
@@ -61,6 +61,14 @@ def destroy
     redirect_to admin_projects_path
   end
 
+  def deploy_group_versions
+    before = params[:before] ? Time.parse(params[:before]).to_s(:db) : Time.now.to_s(:db)
```

Comment:
I left it like that as @grosser wrote the function signature, but yea, looking at it again, I don't see any reason not to change it.